src/main/java/duke/Response.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+
+/**
+ * A duke.Response from the application to some user input.
+ */
+public class Response {
+    private String message;
+    private boolean isExitResponse = false;
+
+    /**
+     * Creates a response with the specified String message.
+     *
+     * @param message The duke.Response message.
+     */
+    public Response(String message) {
+        this.message = message;
+    }
+
+    /**
+     * Converts the duke.Response to its String representation.
+     *
+     * @return The String representation of the duke.Response.
+     */
+    @Override
+    public String toString() {
+        return this.message;
```

Comment:
> Avoid unnecessary use of this with fields.
> Use the this keyword only when a field is shadowed by a method or constructor parameter.

This is an advanced rule of the coding standard which you may want to follow. I noticed this in a few other places too.